gradle.properties

```diff
@@ -1,5 +1,5 @@
 kotlin.code.style=official
-org.gradle.jvmargs=-Xmx4g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=10 -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx5g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=20 -Dfile.encoding=UTF-8
```

Comment:
I propose next time do not apply analysis-plugin
because now we don't use it.
I want to see those resource usages necessary

Reply:
Ok, agree for the next incident